crates/net/network-api/src/lib.rs

```diff
@@ -148,6 +148,9 @@ pub trait Peers: PeersInfo {
     /// Disconnect an existing connection to the given peer using the provided reason
     fn disconnect_peer_with_reason(&self, peer: PeerId, reason: DisconnectReason);
 
+    /// reconnect a connection to the given peer
+    fn reconnect_peer(&self, addr: SocketAddr, peer: PeerId);
```

Comment:
what's the difference between reconnect and just connect?

Reply:
was wondering the same, let's just call it `connect_peer`